src/plugins/intel_gpu/src/plugin/transformations_pipeline.cpp

```diff
@@ -194,6 +194,8 @@ void TransformationsPipeline::apply(std::shared_ptr<ov::Model> func) {
 
         // fuse softmax patterns so that they will not be marked as precision sensitive in ConvertPrecision
         manager.register_pass<ov::pass::SoftmaxFusion>();
+        // decompose MVNs that sre not supported in GPU, so the they will be marked as precision sensitive in ConvertPrecision
+        manager.register_pass<ov::pass::MVN6Decomposition>();
```

Comment:
why do you change it? Not clear. I see it is registered in the common optimization.